Add Loggly supported timestamp field

nywilken opened the pull request:
Thanks for contributing to Logstash! If you haven't already signed our CLA, here's a handy link: https://www.elastic.co/contributor-agreement/

In order for Loggly to parse timestamps for a given log event the
timestamp data must be contained in a timestamp field - not @timestamp.
This change replaces the default @timestamp field of an event with a
timestamp field before posting to Loggly; Addressing Github Issue #12

karmi commented:
Hi @nywilken, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/logstash-plugins/logstash-output-loggly/pull/19.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

nywilken commented:
Hi @karmi thanks for the prompt comment. I went ahead and updated my Github profile and submitted a fix for a rspec bug I introduced in my last commit. The PR is ready for review.

suyograo commented:
@nywilken rebase please?

webmat commented:
Thanks for your contribution and sorry for the enormous delay in getting to this.

This fix was implemented differently in #28, and was released as v5.0.0.
